src/dune_rules/ordered_set_lang.ml

```diff
@@ -213,23 +215,99 @@ let field ?check name =
   in
   Dune_lang.Decoder.field name decode ~default:standard
 
-module Unexpanded = struct
-  type ast = (String_with_vars.t, Ast.unexpanded) Ast.t
+module type Element_intf = sig
```

Comment:
In our convention, the `_intf` suffix is reserved for `.ml` only modules that contain module singatures (whenever we don't want to duplicate the signature in the mli). Module types never have this suffix so this should be called `Element`.

Reply:
Therefore, if if you prefer, you can move `Ordered_set_lang.Element` to `Ordered_set_lang_intf.Element` to avoid writing the module signature twice.